Sources/AblyAssetTrackingCore/Logging/AblyLogHandler+Default.swift

```diff
@@ -0,0 +1,41 @@
+import Foundation
+
+public extension AblyLogHandler {
```

Comment:
The purpose is to align the iOS and Android's api (since for now it looks like it's our source of truth ;) ), and to provide some convenience methods for logging.